### PR TITLE
RATEST-90:AddPatientAppointmentTest including weekends

### PR DESCRIPTION
**Pull request by insookwa**
Added Activation of Weekends on running the test and disabling after the test run . and eliminated ignore 
ticket:https://issues.openmrs.org/browse/RATEST-90?filter=-1

### CODE REVIEW

**Comment by kdaud** [comment]
Waiting for approval to run actions!
cc: @k-joseph

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java [hunk 1]
```diff
@@ -62,12 +61,26 @@ public void addPatientAppointmentTest() throws Exception {
...
     }
+    
+    public void includeWeekends() {
+    	SystemAdministrationPage systemAdministrationPage = new SystemAdministrationPage(page);
+		systemAdministrationPage.activateWeekends("true");
+		homePage.goToPage("index.htm");	
+    }
+    
+    public void removeWeekends() {
+    	homePage.goToPage("index.htm");
```
Comment (k-joseph) [inline ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:75]: home page url should already be index.htm or home.page which should actually load the same thing

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java [hunk 2]
```diff
@@ -62,12 +61,26 @@ public void addPatientAppointmentTest() throws Exception {
 
     @After
     public void tearDown() throws Exception {
+    	removeWeekends();
```
Comment (k-joseph) [inline ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:64]: please use the same indention or spacing as is found in the this same file, consider the next line for-example

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java [hunk 3]
```diff
@@ -62,12 +61,26 @@ public void addPatientAppointmentTest() throws Exception {
...
         deletePatient(patient);
     }
+    
+    public void includeWeekends() {
+    	SystemAdministrationPage systemAdministrationPage = new SystemAdministrationPage(page);
+		systemAdministrationPage.activateWeekends("true");
```
Comment (k-joseph) [inline ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:70]: you will need to resolve the indention here as well

File: ui-tests/src/test/java/org/openmrs/reference/page/SystemAdministrationPage.java [hunk 4]
```diff
@@ -36,4 +42,14 @@ public StylesGuidePage clickOnStylesGuideAppLink() {
         clickOn(STYLES_GUIDE_LINK);
         return new StylesGuidePage(this);
     }
+    
+    public  AdministrationPage activateWeekends(String value) {
+    	clickOn(ADMINISTRATION);
+    	clickOn(MANAGE_GLOBALS);
+    	driver.findElement(EDIT_WEEKENDS).click();
```
Comment (k-joseph) [inline ui-tests/src/test/java/org/openmrs/reference/page/SystemAdministrationPage.java:49]: please check if the value isn't active already in order to set it to that else don't re-active it and if it was initially found active, ensure it's not set even in the before method

**Comment by kdaud** [comment]
@insookwa make sure you configure your IDE to only show your changes. Have you ever had a look at this [wiki](https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse#DeveloperHow-ToSetupAndUseEclipse-HowToConfigureYourIDE) ?

**Comment by insookwa** [comment]
> @insookwa make sure you configure your IDE to only show your changes. Have you ever had a look at this [wiki](https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse#DeveloperHow-ToSetupAndUseEclipse-HowToConfigureYourIDE) ?

Thank you @kdaud . I have already configured my eclipse following that wiki. could this be on Git side

**Comment by kdaud** [comment]
> could this be on Git side

Try to figure it out to ensure only your changes are tracked

**Comment by kdaud** [comment]
Waiting approval to run GitHub actions. cc: @ibacher

**Pull request closed without merging**